File: day-1/1.ts
const text = await Deno.readTextFile("./input.txt");

const result = text
  .split("\n\n")
  .map((elf) =>
    elf
      .split("\n")
      .map((i) => parseInt(i))
      .reduce((a, b) => a + b, 0)
  )
  .sort((a, b) => b - a);

console.log(result[0]);

export {};

File: day-1/1.exs
File.read!("./input.txt")
|> String.split("\n\n")
|> Enum.map(fn(elf)->
  elf |> String.split("\n")
  |> Enum.map(fn(j)-> String.to_integer(j) end)
  |> Enum.reduce(fn(x, acc) -> x + acc end)
end)
|> Enum.sort(&( &1 > &2 ))
|> Enum.at(0)
|> IO.inspect
